### PR TITLE
Getting product using Symfony autowiring

### PR DESCRIPTION
**Pull request by abdounikarim**
Hello,

I add a code block to get product using Symfony autowiring and DependencyInjection.
Do you think it's useful ?

Cheers 😉

### CODE REVIEW

File: doctrine.rst [hunk 1]
```diff
@@ -522,6 +522,33 @@ be able to go to ``/product/1`` to see your new product::
...
+                'No product found for id '.$id
+            );
+        }
+
+        return new Response('Check out this great product: '.$product->getName());
+
+        // or render a template
+        // in the template, print things with {{ product.name }}
+        // return $this->render('product/show.html.twig', ['product' => $product]);
+    }
```
Comment (OskarStark) [inline doctrine.rst]:
```rst
Another possibility is to use the ``ProductRepository`` injected by the dependency injection container::

    // src/Controller/ProductController.php
    // ...
    use App\Repository\ProductRepository;

    /**
     * @Route("/product/{id}", name="product_show")
     */
    public function show($id, ProductRepository $productRepository)
    {
        $product = $productRepository
            ->find($id);
        
        // ...
    }
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Nice addition, but I would shorten the example a bit

File: doctrine.rst (hunk 1 above)
Comment (abdounikarim) [reply doctrine.rst]: Hello @OskarStark , that's just what i did in the code ?

Comment (OskarStark) [reply doctrine.rst]: Yes, but it is too long and verbose, can you please use exactly my code block?

Comment (abdounikarim) [reply doctrine.rst]: done

File: doctrine.rst [hunk 2]
```diff
@@ -523,10 +523,10 @@ be able to go to ``/product/1`` to see your new product::
         // return $this->render('product/show.html.twig', ['product' => $product]);
     }
     
-This in an alternative example using Symfony's autowiring and Dependencyinjection : 
-    
+Another possibility is to use the ``ProductRepository`` injected by the dependency injection container::
```
Comment (OskarStark) [inline doctrine.rst]: autowiring should explicitly mentioned here as this is responsible, not the DIC standalone, to use it this way

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: doctrine.rst (hunk 2 above)
Comment (abdounikarim) [reply doctrine.rst]: done

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you and merry Christmas 🎄

**Comment by abdounikarim** [comment]
> Thank you and merry Christmas 🎄

thanks you @OskarStark , merry christmas too 😉

**Comment by javiereguiluz** [comment]
@abdounikarim we finally merged this! I'm sorry for being so late. Thank you!

**Pull request merged**